frontend/lib/js/form-components/InputRange.js

```diff
@@ -70,6 +84,11 @@ const InputRange = (props: PropsType) => {
           min: value ? value.min : null,
           max: value ? value.max : null,
           [type]: nextValue
+        },
+        {
+          min: props.input.formattedValue ? props.input.formattedValue.min : null,
+          max: props.input.formattedValue ? props.input.formattedValue.max : null,
+          [type]: formattedValue
```

Comment:
I don't know why -- but when I was testing, the x-Buttons inside the input elements didn't have the expected effect. 

When resetting in exact mode, nothing happens, when in range-mode, only the formattedValue is reset but not the underlying raw value.

Reply:
Should be correct now